addonlibs/comm/MessageSys/model/MessageSys.ovm

```diff
@@ -1,6 +1,24 @@
 #include "ov.ovm"
 #include "ksapi.ovm"
 
+/*
+	*
+	*   Chair of Process Control Engineering
```

Comment:
I miss the city in this line :-)
Dresden is calling itself also "Chair of Process Control Engineering"